delete.ps1

```diff
@@ -0,0 +1,161 @@
+############################################
+# HelloID-Conn-Prov-Target-Facilitor-Delete
+# PowerShell V2
+############################################
+
+# Enable TLS1.2
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12
+
+#region functions
+function Resolve-FacilitorError {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [object]
+        $ErrorObject
+    )
+    process {
+        $httpErrorObj = [PSCustomObject]@{
+            ScriptLineNumber = $ErrorObject.InvocationInfo.ScriptLineNumber
+            Line             = $ErrorObject.InvocationInfo.Line
+            ErrorDetails     = $ErrorObject.Exception.Message
+            FriendlyMessage  = $ErrorObject.Exception.Message
+        }
+
+        try {
+            #  Collect ErrorDetails
+            if ($ErrorObject.Exception.GetType().FullName -eq 'Microsoft.PowerShell.Commands.HttpResponseException') {
+                $httpErrorObj.ErrorDetails = $ErrorObject.ErrorDetails.Message
+
+            }
+            elseif ($ErrorObject.Exception.GetType().FullName -eq 'System.Net.WebException') {
+                if ($null -ne $ErrorObject.Exception.Response) {
+                    if ([string]::IsNullOrEmpty($ErrorObject.ErrorDetails.Message)) {
+
+                        $streamReaderResponse = [System.IO.StreamReader]::new($ErrorObject.Exception.Response.GetResponseStream()).ReadToEnd()
+                        if ($null -ne $streamReaderResponse) {
+                            $httpErrorObj.ErrorDetails = $streamReaderResponse
+                        }
+                    }
+                    else {
+                        $httpErrorObj.ErrorDetails = $ErrorObject.ErrorDetails.Message
+                    }
+                }
+            }
+            $errorDetailsObject = ($httpErrorObj.ErrorDetails | ConvertFrom-Json)
+            $httpErrorObj.FriendlyMessage = "$($errorDetailsObject.error.message)"
+
+        }
+        catch {
+            $httpErrorObj.FriendlyMessage = $httpErrorObj.ErrorDetails
+        }
+        Write-Output $httpErrorObj
+    }
+}
+#endregion
+
+try {
+    # Verify if [aRef] has a value
+    if ([string]::IsNullOrEmpty($($actionContext.References.Account))) {
+        throw 'The account reference could not be found'
+    }
+
+    $headers = @{
+        'Content-Type'        = 'application/json; charset=utf-8'
+        Accept                = 'application/json; charset=utf-8'
+        'X-FACILITOR-API-KEY' = $actionContext.Configuration.APIKey
+    }
+
+    Write-Verbose "Verifying if a Facilitor account for [$($personContext.Person.DisplayName)] exists"
```

Comment:
Write-Verbose moet Write-Information zijn.

Reply:
Ook even controleren bij de andere files (CTRL + SHIFT + F) in VScode

Reply:
Zal wel een copy paste dingetje zijn zal ze vervangen